### PR TITLE
Merge 2.6 in master

### PR DESCRIPTION
**Pull request by yguedidi**
As #7140 is the only remaining PR not forward ported to master, I'm trying a proper merge of 2.6 in master so that all fix commits become part of master starting from 2.6.8 :)
There were no complex conflicts to resolve, proof is that the diff is similar to the one of #7140 

- [x] #7143
- [x] redo the PR after 2.6.8 release

### CODE REVIEW

**Comment by yguedidi** [comment]
@j0k3r @Kdecherf @nicosomb sorry I requested the review here too early, I forgot about #7143 that should be merged first 😓

**Review by j0k3r — APPROVED** [review]

**Comment by yguedidi** [comment]
thanks for the approve @j0k3r ! I plan to merge this tonight so that I can rebase #7137 and unblock the 2.7.0 release :)
@nicosomb @Kdecherf if you want to have a look before tonight

**Comment by yguedidi** [comment]
@Kdecherf friendly reminder for a second review :)

**Review by nicosomb — APPROVED** [review]

**Pull request merged**
